config/prod/iwanna-ubuntu-instance.yaml

```diff
@@ -0,0 +1,49 @@
+# Provision an EC2 instance
+template_path: remote/managed-ec2-ubuntu-v2.j2
```

Comment:
Is the j2 a typo?

Reply:
no, because the template changed from being a yaml file to a jinja file.